tests/fixtures/HLS-VI_stac_item.json

```diff
@@ -0,0 +1,109 @@
+{
+  "type": "Feature",
+  "stac_version": "1.0.0",
+  "id": "HLS-VI.L30.T06WVS.2024120T211159.v2.0",
+  "properties": {
+    "start_datetime": "2024-05-08T16:25:12.736230Z",
+    "end_datetime": "2024-05-08T16:25:12.736230Z",
+    "platform": "landsat-8",
+    "instruments": ["oli"],
+    "eo:cloud_cover": 6.0,
+    "proj:transform": [30.0, 0.0, 399960.0, 0.0, -30.0, 7200000.0, 0.0, 0.0, 1.0],
+    "proj:shape": [3660, 3660],
+    "proj:epsg": 32606,
+    "view:sun_azimuth": 168.41547441,
+    "view:azimuth": 179.72857034,
+    "sci:doi": "10.5067/HLS/HLSL30_VI.002",
+    "datetime": "2024-05-08T16:25:12.736230Z"
+  },
+  "geometry": {
+    "type": "MultiPolygon",
+    "coordinates": [
+      [
+        [
+          [-149.114568, 64.909135],
+          [-146.793616, 64.923996],
+          [-146.800894, 63.938713],
+          [-149.040061, 63.924491],
+          [-149.114568, 64.909135]
+        ]
+      ]
+    ]
+  },
+  "links": [
+    {
+      "rel": "self",
+      "href": "https://data.lpdaac.earthdatacloud.nasa.gov/lp-prod-public/HLSL30.020/HLS-VI.L30.T06WVS.2024120T211159.v2.0/HLS-VI.L30.T06WVS.2024120T211159.v2.0_stac.json",
```

Comment:
@madhuksridhar can help me here. I will change the string values.

Reply:
I suggest that we update the href values to replace `HLS` to `HLS-VI` since this is the logical naming convention for VI. but note that we are awaiting confirmation from lpdaac for the exact url and may have to revise this if needed.

Reply:
I will just change it to `HLS-VI` for now and we can update it once we get further confirmation.